.markdownlint.json

```diff
@@ -1,6 +1,10 @@
 {
     "default": true,
     "MD013": false,
+    "MD024": false,
```

Comment:
any reason for 24 to be here?

Reply:
This is the multiple heading warning, which happens all the time because of headings which are actually tab titles.

We can try to find a workaround (maybe @nschonni has an idea), I'd just rather get this PR green and start working from there to remove suppressions as we feel is necessary.

Reply:
Ignore my comment then. Forgot this warning shows up for our tabs.

Reply:
There are some option for restricting it the siblings https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024---multiple-headings-with-the-same-content